wrappers/python/anoncreds/types.py

```diff
@@ -40,25 +40,46 @@ def load(cls, value: Union[dict, str, bytes, memoryview]) -> "CredentialDefiniti
         )
 
     @property
-    def id(self) -> str:
+    def schema_id(self) -> str:
```

Comment:
This seems to drop the "id" property accessor?

Reply:
To my knowledge there isn't actually an id field in CredentialDefinition.
https://github.com/Gavinok/anoncreds-rs/blob/main/src/data_types/cred_def.rs#L39-L48

Feel free to correct me if I am missing something